config/crd/kustomization.yaml

```diff
@@ -1,8 +1,6 @@
 # This kustomization.yaml is not intended to be run by itself,
 # since it depends on service name and namespace that are out of this kustomize package.
 # It should be run by config/
-commonLabels:
-  cluster.x-k8s.io/v1beta1: v1beta1
```

Comment:
This is the only interesting change here I think. Since the existing CRDs are labeled for v1beta1, we need to label only the new CRDs for v1alpha1. Now we have an individual patch file for each CRD because I wasn't motivated to come up with a sleeker solution.